shared-articles/what does dcdd do.md

```diff
@@ -0,0 +1,90 @@
+
+With each connection a node makes, it shares its own connections, like saying "here are all the IP addresses I know of who run dcrd." Each node builds its own list of peers to check and ask, "hey, do any of you have any new blocks?" Full nodes can do this without accepting inbound connections. They still need to maintain a list of peers to query for each outbound connection.
+
+However, imagine if nobody opened up inbound connections through their firewall. Every dcrd node would just continually reach out to the IP addresses in their peers list, asking "Hey, can I talk to you? Do you have new blocks?" And each firewall would just block all traffic. No nodes would be able to talk to each other.
+
+The point is that somebody has to open up their inbound connection to allow their node to be publicly accessible. The more people that do this will increase the security and decentralization of the network. 
+
+## Other nodes in the network
+Decred has a multitude of services that run dcrd nodes in one way or another. Some of these options also include incentives for participation, which in turn encourages users to run a node that fully supports the network. For example, you're incentivised to run a full node so that you don't have to trust a 3rd party. IF you're solo-staking, you're incentivised for your full node to have a high uptime and allow inbound connections to your node.
+
+## Decred Services and dcrd
+
+**Decrediton** can be run either as a fully validating node or in SPV mode. Wallets will never have an open inbound connection. The main reason for this is because wallets usually only run for short periods of time, not as a long-running background process. Allowing other dcrd instances to connect to an instance which only runs intermittently is actually bad for the network, because every instance connected is going to have to find a replacement peer once the wallet is disconnected. Its better if they just connect to a long-lived peer in the first place.
+
+Wallets running in fully validating mode allow for maximum personal privacy and security. On the other hand, SPV mode is great for mobile wallets or when a device has limited resources. 
+
+**Lightning Network Wallets** are run as fully validating nodes and for maximum accessibility can have an open inbound connection. LN operators are also incentivised, to run a full node that supports the network, with the fees from conducting transactions over the Lightning Network.
+
+**Voting Service Providers (VSP)** are run as fully validating nodes and for maximum accessibility can have an open inbound connection. VSP operators are also incentivised with the fees for conducting ticket transactions. For the best performance, these nodes should run constantly 24/7/365 and have +4 instances to increase reliability.
```

Comment:
> Voting Service Providers (VSP) are run as fully validating nodes and for maximum accessibility can have an open inbound connection.

I think this mixes up the two services, VSP and fully-validating node.

A VSP server needs reliable *RPC connection* to a dcrd node. In theory a VSP operator may run no dcrd nodes at all and use someone else's dcrd *if* that dcrd's admin allows it. In practice though it is cheap and convenient to run a dcrd besides your VSP server and not depend on external services. And as you say, that dcrd may or may not accept inboud connections. What matters is that the VSP API is publicly accessible, but its dcrd instance is an internal "implementation detail" and may be private.